### DISCUSSION TITLE
Master process starts with Nice=-19

### DISCUSSION THREAD

**Discussion by solracsf** [juicedata/juicefs#2312] [General]
Is it possible to override this behavior?
Even when starting the mount trough a `systemd` service with `Nice = 0` it is override.

![image](https://user-images.githubusercontent.com/12234510/175790375-36b14165-cd7c-478e-bcbd-4a19ac7fd617.png)

**Comment by qingyunha** [comment]
It's a hardcode. 
https://github.com/juicedata/juicefs/blob/f0c9b0d3ee7339b63cadaa8f39ed8d2e1ff08d49/pkg/fuse/fuse.go#L418

**Comment by davies** [comment]
Filesystem is a critical service, and most of them running in kernel space. JuiceFS run in user land, it should have higher priority comparing to others.

In a rare case that you need to use a different priority, please for it and change the hardcode one.

**Comment by solracsf** [comment]
I'll go with `renice -n 0 $(pidof juicefs)`
